docs/DeveloperGuide.md

```diff
@@ -334,27 +334,50 @@ This is to prevent `SortCommand` from taking on more responsibilities (Separatio
 contains each of the meetings related to each other stored in an ```ObservableList```. The ``` MeetingManager ``` is
 used to manage any operations that require viewing or sorting of meetings from the ```MeetingList``` class.
 
-#### What designs were considered:
+The operations for adding and deleting meeting are handled by `AddMeetingCommand` and `DeleteMeetingCommand`, which are supported by `AddMeetingCommandParser` and `DeleteMeetingCommandParser` respectively.
+
+1. The user enters `meeting-add 2 d/Meet for practical exam s/20/04/2024 15:00` to add a meeting or `meeting-delete 1 i/1` to delete a meeting.
+2. `Logic Manager` receives the user input which is parsed by `StaffConnectParser`.
+3. After splitting the user input into `commandWord` and `arguments` based on the regex pattern of the user input, the `StaffConnectParser` invokes the `AddMeetingCommandParser`or`DeleteMeetingCommandParser` based on the `commandWord`. Calling the method `parse` with `arguments` as the method arguments, and getting supported by parsing methods from `ParsedUtil`. 
+4. `AddMeetingCommand` or `DeleteMeetingCommand` is created with the parsed values.
+5. `Logic Manager` executes the `AddMeetingCommand` or `DeleteMeetingCommand`, which handles adding/removing meeting from the `Person` respectively and updates the model with the new information.
+
+Below is the sequence diagram for parsing inputs with  `AddMeetingCommandParser` executing `meeting-add 2 d/Meet for practical exam s/20/04/2024 15:00`:
+<br>**Note:**<br> 
+`userInput` represents the value of <br>`meeting-add 2 d/Meet for practical exam s/20/04/2024 15:00`.
+<br> While `arguments` represents the value `2 d/Meet for practical exam s/20/04/2024 15:00`.
+<br>![AddMeetingCommandParser Sequence Diagram](images/AddMeetingParserSequenceDiagram.png)
+<br> Similarly the sequence diagram for parsing inputs with `DeleteMeetingCommandParser` executing `meeting-delete 1 i/1`:
+<br>![DeleteMeetingCommandParser Sequence Diagram](images/DeleteMeetingParserSequenceDiagram.png)
+<br><br>
+After parsing, the commands are executed by the logic manager as show below. (Execute in the diagrams below comes form the logic manager)
+<br> Below is the sequence diagram for adding meeting with  `AddMeetingCommand`:
+<br>![AddMeetingCommand Sequence Diagram](images/AddMeetingSequenceDiagram.png)
+<br> Similarly the sequence diagram for deleting meeting with `DeleteMeetingCommand`:
+<br>![DeleteMeetingCommand Sequence Diagram](images/DeleteMeetingSequenceDiagram.png)
+
+#### Design considerations:
 
 **Aspect: How the meetings are stored :**
 
-* **Current Design:** Store meetings in an ObservableList.
+* **Alternative 1 (current choice):** Store meetings in an ObservableList.
     * Pros: Better segregation of the OOP functionalities, and good integration with the UI ListView.
     * Cons: Larger code complexity.
 
-* **Alternative 1:** Store meetings in a Set.
+* **Alternative 2:** Store meetings in a Set.
```

Comment:
these points were also changed, should be `Current Design` (alt 1) and `Alternative 1` (alt 2)

Reply:
Oh opps coz i was using the web editor to do the merge conflicts, my local repo had issues syncing

Reply:
The merge errors has been fixed!